Cargo.toml

```diff
@@ -80,32 +80,17 @@ jsonrpsee-types = "0.24"
 
 # misc
 cfg-if = "1"
-hashbrown = "0.14.5"
 spin = { version = "0.9.8", features = ["mutex"] }
 derive_more = { version = "1.0", default-features = false }
 
 ## misc-testing
 arbitrary = { version = "1.3", features = ["derive"] }
 rand = "0.8"
 thiserror = "1.0"
-proptest = "1.4"
-proptest-derive = "0.4"
+proptest = "1.5"
+proptest-derive = "0.5"
 tokio = "1"
 
 ## crypto
 c-kzg = { version = "1.0", default-features = false }
 k256 = { version = "0.13", default-features = false, features = ["ecdsa"] }
-
-
-#[patch.crates-io]
-#alloy-consensus = { git = "https://github.com/alloy-rs/alloy", branch = "main" }
-#alloy-network = { git = "https://github.com/alloy-rs/alloy", branch = "main" }
-#alloy-network-primitives = { git = "https://github.com/alloy-rs/alloy", branch = "main" }
-#alloy-rpc-types = { git = "https://github.com/alloy-rs/alloy", branch = "main" }
-#alloy-rpc-types-eth = { git = "https://github.com/alloy-rs/alloy", branch = "main" }
-#alloy-rpc-types-engine = { git = "https://github.com/alloy-rs/alloy", branch = "main" }
-#alloy-eips = { git = "https://github.com/alloy-rs/alloy", branch = "main" }
-#alloy-serde = { git = "https://github.com/alloy-rs/alloy", branch = "main" }
-#alloy-signer = { git = "https://github.com/alloy-rs/alloy", branch = "main" }
-#alloy-provider = { git = "https://github.com/alloy-rs/alloy", branch = "main" }
-#alloy-transport = { git = "https://github.com/alloy-rs/alloy", branch = "main" }
```

Comment:
please keep them, this will become useful again

Reply:
Got it will do